FirebaseRemoteConfig/Tests/SwiftAPI/Resources/README.md

```diff
@@ -0,0 +1,66 @@
+# Remote Config Console API
+
+[`RemoteConfigConsole.swift`](https://github.com/firebase/firebase-ios-sdk/blob/nc-rc-test-api/FirebaseRemoteConfig/Tests/SwiftAPI/RemoteConfigConsole.swift) provides a simple API for interacting with an app's remote config on the Firebase console.
+
+## Local Development Setup
+
+You can start by generating the `FirebaseRemoteConfig` project:
```

Comment:
Think about separating the steps from the explanation.

This document should address two use cases separately:
1. As simple as possible set of steps to start running as quickly as possible without necessarily understanding anything.
2. A separate explanation of why those steps are needed.

Reply:
README has been updated